edb/server/pgcon/pgcon.pyx

```diff
@@ -135,6 +135,11 @@ async def connect(connargs, dbname):
             ignore_data=True,
         )
 
+    await pgcon.simple_query(
```

Comment:
This should go into `connect()` in this file to avoid the extra query roundtrip.

Reply:
Yeah, move it to pgcon.pyx:1192